server/sonar-web/src/main/js/components/store/globalMessages.js

```diff
@@ -54,6 +55,24 @@ const globalMessages = (state = [], action = {}) => {
     }];
   }
 
+  if (action.type === actions.REQUIRE_AUTHENTICATION) {
+    // FIXME l10n
+    return [{
+      id: uniqueId('global-message-'),
+      message: 'Authentication required to see this page.',
+      level: ERROR
+    }];
+  }
+
+  if (action.type === actions.REQUIRE_AUTHORIZATION) {
+    // FIXME l10n
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1134)